src/zed_cpu_ros.cpp

```diff
@@ -168,11 +169,11 @@ class ZedCameraROS {
 			// get config from the left, right.yaml in config
 			camera_info_manager::CameraInfoManager info_manager(nh);
 			info_manager.setCameraName("zed/left");
-			info_manager.loadCameraInfo( "package://zed_cpu_ros/config/left.yaml");
+			info_manager.loadCameraInfo( "package://zed_cpu_ros/config/" + std::to_string(resolution_) + "/left.ini");
```

Comment:
can you change these two configs into an input param as a string?
different cameras might will have different calibration parameters,
so really make this as a param make more sense.
These two calibration is just an example of what people should be expecting.
Eventually they will have to do a calibration.